pkg/disk/json_test.go

```diff
@@ -0,0 +1,127 @@
+package disk
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/rsa"
+	"fmt"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/go-jose/go-jose/v3"
+	"github.com/go-jose/go-jose/v3/cryptosigner"
+	"github.com/go-jose/go-jose/v3/jwt"
+	"github.com/spiffe/go-spiffe/v2/bundle/jwtbundle"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/go-spiffe/v2/svid/jwtsvid"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	jwtBundleFilename = "jwt_bundle.json"
+	jwtSVIDFilename   = "jwt.json"
+)
+
+func TestWriteJWTBundleSet(t *testing.T) {
+	td, err := spiffeid.TrustDomainFromString("example.test")
+	require.NoError(t, err)
+
+	jwtBundle := jwtbundle.New(td)
+	jwtBundleSet := jwtbundle.NewSet(jwtBundle)
+	require.True(t, jwtBundleSet.Has(td))
```

Comment:
NIT: sound to me like this validation is out of scope, it is validating a go-spiffe validation, anyway, you can keep it if you prefer :D

Reply:
Removed